apps/web/src/lib/components/home/Hero.svelte

```diff
@@ -1,25 +1,53 @@
-<section class="h-[80vh] bg-hero bg-cover bg-center bg-no-repeat px-4">
-	<div
-		class="container mx-auto flex h-full flex-col items-center justify-center"
-	>
-		<h1 class="h1 z-10 text-center text-5xl font-bold text-white">
-			Welcome to Freedom in Christ Church.
-		</h1>
-		<h2 class="h2 my-8 text-center text-2xl text-white md:w-[50rem]">
-			A Christ-centred, Holy Spirit-led community committed to the building of
-			strong relationships and planting of healthy reproducing churches.
-		</h2>
-		<div class="my-8 flex gap-8">
-			<a
-				href="/about"
-				class="bg-surface-200-700-token text-token btn btn-xl"
-				data-sveltekit-preload-data="hover">I'm new</a
+
+<section style="background-image: url('/images/youngPeopleImage.jpg');" class="flex h-[30vh] sm:h-[90vh] bg-contain bg-no-repeat sm:bg-cover sm:bg-gradient-to-t sm:from-blue-950 to-green-800 items-center justify-center ">
+	
+	<div style="background-image: url('/images/youthImage.jpg');" class="sm:rounded-2xl sm:bg-gradient-to-t sm:from-blue-950 to-green-800 mt-40 sm:mt-[480px] mx-auto bg-contain bg-no-repeat flex h-[420px] sm:h-auto p-4 items-center justify-around ">
+		<div
+			class="container mx-10 flex h-full flex-col items-center justify-center"
+		>
+			<h1
+				class="tracking-widest mt-80 sm:mt-[300px] h1 mb-10 flex flex-col rounded-lg sm:border sm:bg-green-800 p-2 text-center text-5xl font-bold text-white sm:shadow-md sm:shadow-white"
 			>
-			<a
-				href="/sermons"
-				class="bg-surface-200-700-token text-token btn btn-xl"
-				data-sveltekit-preload-data="hover">Sermons</a
+				Freedom in Christ <span class=" text-white">Church</span>
+			</h1>
+			<h2
+				class="h2 hidden sm:flex top-52 sm:mt-[30px] border-b border-slate-700 text-center text-2xl text-white"
 			>
+				A Christ-centred, Holy Spirit-led community -
+			</h2>
+
+			<h2
+				class="h2 hidden sm:flex top-52 border-b border-slate-700 text-center text-2xl text-white"
+			>
+				Committed to the building of strong relationships -
+			</h2>
+			<h2
+				class="h2 hidden sm:flex top-52 border-b border-slate-700 text-center text-2xl text-white"
+			>
+				and planting of healthy reproducing churches.
+			</h2>
 		</div>
+		
+	</div>
+</section>
+<section class="mt-44 bg-slate-400 p-2">
+	<div class="my-10 flex items-center justify-center gap-10">
+		<a
+			href="/about"
+			class=" text-token btn btn-xl border bg-green-800"
+			data-sveltekit-preload-data="hover">I'm new</a
+		>
+		<a
+			href="/sermons"
+			class="bg-surface-200-700-token text-token btn btn-xl border"
+			data-sveltekit-preload-data="hover">Sermons</a
+		>
 	</div>
 </section>
+<div
+	class="sticky top-0 z-20 flex justify-center border-y bg-warning-800 p-2 shadow-2xl shadow-green-700"
+>
+	<button>
+		<h1 class="text-4xl tracking-widest">Schedule your first visit!</h1>
```

Comment:
There should only be one `<h1>` on a page at a time, you can remove this and just add the styling to the `<button>` element